cloudevents/tests/test_pydantic_conversions.py

```diff
@@ -146,12 +176,15 @@ def test_from_dict():
         ),
         "type": "dummy.type",
     }
-    assert from_dict(given).dict() == given
+    assert cloudevents_implementation["from_dict"](given).dict() == given
 
 
+@pytest.mark.xfail  # Pydantic V2 doesn't support function params
```

Comment:
As the comment says, pydantic V2 doesn't accept kwargs on the json() method anymore.

I haven't found any usage of event.json() in this SDK, I wonder if we should test this at all.